app/components/scheduler/unscheduled-session.js

```diff
@@ -0,0 +1,21 @@
+import Component from '@ember/component';
+
+export default Component.extend({
+  didRender() {
```

Comment:
Use `didInsertElement`. Because a `didRender` hook gets called everytime there is a re-render. Which would mean, we'd be doing all of this multiple times for no reason. Just put it in didInsertElement. Should be okay.

Reply:
okay cool